File: Lab_2/src/Romb.java
public class Romb extends Czworokat
{

    Romb(double bok1, double kat)
    {
        super(bok1, bok1, bok1, bok1, kat);
    }

    @Override
    public double count_Pole()
    {
        pole = Math.pow(bok1, 2) * Math.sin(Math.toRadians(kat));
        pole = Math.round(pole * 10.0) / 10.0;
        return pole;
    }

    @Override
    public String get_name()
    {
        return "Romb";
    }
}

File: Lab_2/С++/Romb.cpp
#include <Romb.h>

    Romb::Romb(double bok1, double kat) : Czworokat (bok1, bok1, bok1, bok1, kat) { }

    double Romb::count_Pole()
    {
        pole = pow(bok1, 2) * sin(kat*3.14159/180);
        pole = round(pole * 10.0) / 10.0;
        return pole;
    }

